### PR TITLE
Added script for jquery-labelauty

### PR DESCRIPTION
**Pull request by prashantkhoje**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package: jquery-labelauty
+# Version: v1.1.4
+# Source repo: https://github.com/fntneves/jquery-labelauty 
+# Tested on: RHEL v8.5
+# Language: PHP
```
Comment (seth-priya) [inline j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh:8]: this seems to be a node package?

File: j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```
Comment (seth-priya) [inline j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh:1]: can we use "#!/bin/bash -e" for consistency and remove "set -ex" on line #21?

File: j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# -----------------------------------------------------------------------------
+#
+# Package: jquery-labelauty
+# Version: v1.1.4
+# Source repo: https://github.com/fntneves/jquery-labelauty 
+# Tested on: RHEL v8.5
```
Comment (seth-priya) [inline j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh:7]: can we make this "UBI 8.5" for consistency with the build script name?

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh (hunk 2 above)
Comment (prashantkhoje) [reply j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh]: done

File: j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh (hunk 1 above)
Comment (prashantkhoje) [reply j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh]: updated

File: j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh (hunk 3 above)
Comment (prashantkhoje) [reply j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh]: done

**Review by seth-priya — APPROVED** [review]

**Pull request merged**